packages/data-models/constants.ts

```diff
@@ -48,6 +48,11 @@ const APP_LANGUAGES = {
   default: "gb_en",
 };
 
+const APP_SKINS = {
+  /** Default skin used if none other is explicitly set */
+  default_skin_name: "weekly_workshop",
```

Comment:
nit(blocking)
This feels like the wrong place to be defining plh-specific content. Also by making this a default it will have an immediate knock-on for other apps.

If we really need to force some overrides just for plh then lets do it from the plh deployment config files instead

Reply:
I think you're right to flag here that we don't want to be defining plh-specific content at the level of the default app constants. But I think this might be a feature of my current naming convention for skins: I've renamed the default skin from "default" to "weekly_workshop", which is a descriptive name to contrast it with the second, "modular" skin. (Likewise for themes, we currently have "blob" and "professional"). 

Assuming that all apps/deployments need a default skin, it makes sense to specify a skin which will be the default skin if no default skin is explicitly specified by the app deployment config. And this seems like the place to define that. If so, then currently our only option for the default skin is one of the two existing skins. 

I can see a few ways to fix this:
1. Require a deployment config to specify a default skin, meaning we don't have to provide one at the default app constants level
2. Change the name of the "weekly_workshop" skin to "default" (or "base" or equivalent)
3. Duplicate the "weekly_workshop" skin and name this third skin "default", to be used as the default skin when none other is specified. These two skins would initially be identical, but could diverge in the future. This would allow the plh app to switch between two descriptively named skins, and apply template overrides based on conditions relating to either skin. But it would also mean that other apps would have the same default skin that they do currently, but without using a plh-specific name

It's worth saying that "weekly_workshop" and "modular" aren't necessary plh-specific names. Our default skin, in combination with our default templates, is configured to display content in the form of weekly workshops. But it's true that the skin itself doesn't contain that information, it's the default templates that do: the skin only specifies not to apply any skin-specific template overrides.

Reply:
I would argue that neither `weekly_workshop` or `modular` are descriptive outside the context of the plh app. If we are thinking of what a _default_ skin might be you can look at how ionic [templates ](https://ionicframework.com/docs/v3/cli/starters.html)it's apps - you basically get a choice of things like _blank_ , _sidebar_ or _tabs_ which are all ways to decide what sidebar/header/footer options are available (docs link a bit outdated but main idea is there). 

So yes, you are right that we are lacking a real _default_ option - for that I'd probably probably suggest we stick with a sidebar-enabled configuration, which will impose the need for matching header and sidebar templates (fine for now, in the future we can include a stock/placeholder template to at least include a header with the sidebar toggle button). If we want a more descriptive plh option name then we extend this default template and rename (even if changing no other options) - but my hunch is there will be some minor changes between what we eventually settle on for plh defaults and what we decide makes most sense for general defaults. 

So I think this is most in-line with options (2) and (3) above - we change the name back to default, then extend/duplicate to create a new weekly_workshops skin, which as you say will start identical but diverge over time. Importantly as it diverges it won't break other apps. The weekly workhsops skin can be set as the default skin for all plh deployment configs